kernelci/db/kernelci_api.py

```diff
@@ -103,6 +103,20 @@ def get_root_node(self, node_id):
         resp = self._get('/'.join(['get_root_node', node_id]))
         return json.loads(resp.text)
 
+    def get_completed_event(self, node_id, hours=0, minutes=0, seconds=0):
+        """Get an event when all child nodes are complete of a given node"""
+        resp = self._get(f'trigger_completed_event/{node_id}?wait_time_hours=\
+{hours}&wait_time_minutes={minutes}&wait_time_seconds={seconds}')
+        return resp.json()
+
+    def get_child_nodes_from_event(self, event):
+        """Get child nodes from the event"""
+        return event.data['nodes']
+
+    def get_regressions(self, regression_name):
+        resp = self._get('?'.join(['regressions', 'name='+regression_name]))
```

Comment:
Please, see my previous comment about using `params`.

Please remember about spaces around operators `'name='+regression_name` -> `'name=' + regression_name`

Reply:
Added whitespace around the operator.